pkg/platform/platform_test.go

```diff
@@ -0,0 +1,31 @@
+package platform
+
+import (
+	"testing"
+)
+
+func TestAppendVersionTagUninitializedMap(t *testing.T) {
+	var f map[string]string
+
+	AppendVersionTag(&f)
+
+	if len(f) == 0 {
+		t.Fatalf("version tag should be appended to uninitialized map")
```

Comment:
Nit, for consistency

```suggestion
		t.Fatalf("should append version tag to uninitialized map")
```

Reply:
Thanks.